src/components/TodoList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Todo } from './Todo';
+
+export const TodoList = ({ todos }) => (
+  <ul>
+    <h1>TodoList</h1>
+    {todos.map(todo => (
+      <li key={todo.id} className="todo">
+        <Todo todo={todo} />
```

Comment:
If you use spread operator here, you will be able to easily destructure `title`, `completed` and `user` in `Todo` component.

```suggestion
        <Todo todo={...todo} />
```